ssl/t1_lib.c

```diff
@@ -2408,8 +2408,7 @@ static int ssl_scan_clienthello_tlsext(SSL *s, unsigned char **p,
                 goto err;
             if (!tls1_save_sigalgs(s, data, dsize))
                 goto err;
-        } else if (type == TLSEXT_TYPE_status_request) {
-
+        } else if (type == TLSEXT_TYPE_status_request && !s->hit) {
```

Comment:
do we need to increment ```data``` by ```size``` if we're skipping processing?

Reply:
No, it does that at the end of the loop

Reply:
That will teach me, only looking at the top of the loop...